src/Components/HomePage/HeaderScreen.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import { Container, Button } from 'react-bootstrap';
+import { Container, Button, Row, Col, Card } from 'react-bootstrap';
```

Comment:
The security check failed for `Card was never used`

Reply:
Fixed.